frontend/src/MyContext.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+//import PHASE from "./App";
+//import WaitingRoom from "./WaitingRoom";
+//import WelcomeMenu from "./WelcomeMenu";
+export const MyContext = React.createContext();
+
+export const PHASE = {
+  START: 0,
+  WAITING_ROOM: 1,
```

Comment:
waiting room should be 0

Reply:
Sure? I changed the 0 key to be WELCOME_SCREEN from start as it should be. I think it is now ok